### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JanZachmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,44 +2,51 @@
 
 # (e)mbedded (l)inux (b)uilding (b)locks - local development environment
```
Comment (JoergZeidler) [inline README.md:3]: in general, we should remove the "local" info of all environment

File: README.md [hunk 2]
```diff
@@ -2,44 +2,51 @@
...
-This environment contains a concourse CI server, a docker registry and a MinIO server that canbe used for local development.
+## Local Concourse CI environment
 
-The environment can be started using dobi.
+This environment contains a Concourse CI server, a docker registry and a MinIO server that can be used for local development.
 
+The environment can be started using dobi:
 ```sh
-./dobi.sh dev-environment-concourse-start
+./dobi.sh dev-environment-concourse-start ?????
```
Comment (JoergZeidler) [inline README.md:26]: the "dev-environemt-" prefix of all commands should be removed in the readme

File: README.md [hunk 3]
```diff
@@ -116,18 +120,17 @@ docker run --rm -it elbb/registry-cleanup -address http://${DOCKER_HOST_IP}:5000
 ## Local (Conan) Artifactory Community Edition for C/C++
 
 If you want to use or deploy your own conan packages in your concourse pipeline you need a reachable "Artifactory Community Edition for C/C++". This Toolset contains a preprovisioned one with default credentials. Don't use it for production purposes!
-You can start the local "Artifactory Community Edition for C/C++" via
+You can start the local "Artifactory Community Edition for C/C++" from the root directory via
```
Comment (JoergZeidler) [inline README.md:123]: maybe the info that "start from root directory" should be in a general section available

**Review by JoergZeidler — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JanZachmann) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (JanZachmann) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (JanZachmann) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -1,45 +1,52 @@
...
 
 -   [docker](https://docs.docker.com/install/)
--   [dobi](https://github.com/dnephin/dobi) (downloaded if not in `PATH`)
-
-## Local concourse CI environment
+-   [docker compose](https://docs.docker.com/compose/install/)
+-   [dobi](https://github.com/dnephin/dobi): **Always use dobi.sh from root of this directory**. If dobi isn't installed or not in `PATH` yet it will be downloaded and installed automatically by first usage of dobi.sh.
 
-This environment contains a concourse CI server, a docker registry and a MinIO server that canbe used for local development.
+## Local Concourse CI environment
```
Comment (JoergZeidler) [inline README.md:20]: I had meant in my previous comment that the term "local" could be removed from the document in its entire context, as we use these environments on a server, for example.

**Review by JoergZeidler — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (JanZachmann) [reply README.md:20]: I know. There are 31 occurrences of the word "local" in the readme. I'm not sure if it is correct to remove them all. Feel free to remove the ones you think that can be removed.

Comment (JoergZeidler) [reply README.md:20]: We should leave it as it is.

**Review by JoergZeidler — APPROVED** [review]

**Pull request merged**